packages/big-design/src/components/ButtonGroup/styled.tsx

```diff
@@ -6,6 +6,7 @@ import { FlexItem } from '../Flex';
 
 interface StyledButtonProps {
   borderRadius?: boolean;
+  isVisible?: boolean;
```

Comment:
So I don't think we need to add an `isVisible` prop here 🤔  Since there is an absolute positioning happening [here](https://github.com/bigcommerce/big-design/blob/7ce0bbee8b686bd9b6c49333a3a19a084c6ca206/packages/big-design/src/components/ButtonGroup/styled.tsx#L53), all we should need to do is add a huge `right` value:

![Screen Shot 2022-03-08 at 18 31 56](https://user-images.githubusercontent.com/10539418/157349403-adf8f9e3-fd4f-43a8-a979-d2683726cc0f.png)

From what I gathered, we need the button to still be in the dom, but hidden away from everything.

Reply:
Why do we need the ellipsis button to still be in the Dom?

Reply:
It doesn't just apply to the ellipsis, but the other buttons as well. They need to be in the DOM, but hidden from the user. If you use `display:none` the `ref` is set to `undefined` thus we can't do anything with it.